chatkit-android/build.gradle

```diff
@@ -45,28 +45,23 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     if (rootProject.properties.containsKey('pusher_platform_local')) {
         api project(':pusher-platform-android')
     } else {
         api "com.pusher:pusher-platform-android:$pusher_platform_version"
     }
```

Comment:
I know that's an existing issue but thought to use this opportunity to share the finding that this declaration is not needed in chatkit-android — the build is passing without it.

Reply:
Have removed - it'd be nicer if there was a way to automatically see which dependencies weren't actually being used!

Reply:
It's not removed:
```groovy
    if (rootProject.properties.containsKey('pusher_platform_local')) {
        api project(':pusher-platform-android')
    } else {
        api "com.pusher:pusher-platform-android:$pusher_platform_version"
    }
```
is still here

Reply:
Looks like we can't remove this because [AndroidChatkitDependencies](https://github.com/pusher/chatkit-android/blob/master/chatkit-android/src/main/kotlin/com/pusher/chatkit/AndroidChatkitDependencies.kt) depends on it.